app/views/components/_usa-banner.html.erb

```diff
@@ -6,11 +6,11 @@
           <img class="usa-banner__header-flag" src="<%= image_path("us_flag_small.png") %>" alt="U.S. flag">
         </div>
         <div class="grid-col-fill tablet:grid-col-auto">
-          <p class="usa-banner__header-text">An official website of the United States government</p>
-          <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
+          <p class="usa-banner__header-text"><%= t  'header.official' %></p>
```

Comment:
In the future, feel free to split the html onto separate lines, which can make future diffs easier to read and a bit cleaner.

eg:

```
<p class="usa-banner__header-text">
  <%= t  'header.official' %>
</p>
```

Just style preference.

Reply:
double whitespace here after `t`